application/logger.py

```
@@ -0,0 +1,128 @@
+            fpath=self.current_log_file,
+            mode=mode,
+            enable_compression=enable_compression,
+            timestamp_format=timestamp_format,
+            loglevel_timeout=loglevel_timeout,
+        )
+    
+    def _generate_log_filename(self) -> Path:
+        """
+        Generate a log filename with the timestamp and prefix.
+        """
+        start_time = datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)